po/extra/meson.build

```diff
@@ -1,5 +1 @@
-i18n.gettext('extra',
-    args: '--directory=' + meson.source_root(),
-    preset: 'glib',
-    install: false,
-)
+i18n.gettext('extra', args: '--directory=' + meson.project_source_root(), preset: 'glib', install: false)
```

Comment:
Same as before, I think this makes things a bit harder to read at a glance. I'm not sure there's a benefit to compacting it like this